src/main/java/com/exercise/basic/string/Bai7.java

```diff
@@ -0,0 +1,26 @@
+package com.exercise.basic.string;
+
+public class Bai7 {
+
+    public int findSubstring(final String str1, final String str2) {
+        if (str2.trim().length() == 0 || str1.trim().length() == 0) {
```

Comment:
Bỏ lệnh này xem khác không

Reply:
Nó sẽ sai kết quả anh " "anh tuan".indexOf(""));" -> 0 , """.indexOf("anh tuan")" -> -1

Reply:
Kết quả đó đúng mà

Reply:
Em thử check hàm build in xem nó ra kết quả ra sao

Reply:
em hiểu rồi anh. Fixed